### PR TITLE
update

### LINKED ISSUE #7: Intervocalic [y] seem to be occurences of [j]
I also found many [y] here that look like [j], for example:

```
Chayahuita-173_ripe-1,,Chayahuita,173_ripe,kayaɾin,kayaɾin,k a y a ɾ i n,,hart2008a,,,^k a y a ɾ i n$,Chayahuita,kayaɾin,cayarin
Miskito-175_root-1,,Miskito,175_root,plamaya,plamaya,p l a m a y a,,heath1953a,,,^p l a m a y a$,Miskito,plamaya,plamaya
Yuracare-175_root-1,,Yuracare,175_root,boyo,boyo,b o y o,,cueva1893a,,,^b o y o$,Yuracare,boyo,boyo
Yuracare-177_rotten-1,,Yuracare,177_rotten,yoyonyo,yoyonyo,y/j o y o n y o,,cueva1893a,,,^y o y o n y o$,Yuracare,yoyonyo,yoyonyo
Yuracare-180_run-1,,Yuracare,180_run,ayaktai,ayaktai,a y a k t a i,,cueva1893a,,,^a y a k t a i$,Yuracare,ayaktai,a-yacta-i
Yuracare-181_sad-1,,Yuracare,181_sad,ayenyei,ayenyei,a y e n y e i,,cueva1893a,,,^a y e n y e i$,Yuracare,ayenyei,a-yenye-i
Miskito-184_sand-1,,Miskito,184_sand,kabo_auya,kabo_auya,k a b o + a u y a,,heath1953a,,,^k a b o _ a u y a$,Miskito,kabo auya,kabo auya
```

I checked for the language Miskito, and it does not seen to have [y] but has [j], which seems to confirm the intuition that the orthography profile needs to be updated.

This is related to these two issues:

https://github.com/lexibank/transnewguineaorg/issues/14#issuecomment-782874114
https://github.com/lexibank/walkerarawakan/issues/12#issuecomment-782873959

### PR DESCRIPTION
**Pull request by MuffinLinwist**
Standard update with fixes on ortho-profiles to close #7.

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -5,4 +5,4 @@ Name | GitHub user | Description| Role
 Sacha Beniamine | @XachaB | | Other
 Erben Johansson | | publication author | Author
```
Comment (chrzyki) [inline CONTRIBUTORS.md:6]: We should probably include the full name here like we typically do, i.e. "Niklas Erben Johansson"

**Review by FredericBlum — COMMENTED** [review]
Bar the comment from @chrzyki , this looks good

File: CONTRIBUTORS.md (hunk 1 above)
Comment (MuffinLinwist) [reply CONTRIBUTORS.md:6]: thanks :)

**Comment by MuffinLinwist** [comment]
I adopted the comment from @chrzyki. So, now everything should be ready for release.

**Review by chrzyki — APPROVED** [review]

**Pull request merged**